### PR TITLE
Add docker dev setup

### PR DESCRIPTION
**Pull request by lsetiawan**
## Overview

This PR adds docker development setup!

### CODE REVIEW

**Comment by emiliom** [comment]
Thanks @lsetiawan !! I'm not in a position to truly assess this (as a docker newbie), but I'm sure it's good, or at least 98% of the way there. So, merging.

**Comment by lsetiawan** [comment]
Now as you develop code, it will reflect the changes on the wofpy server right away :)

File: docker_dev/scripts/setup-mysql.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+cd $APP_PATH
+mkdir tmp && cd tmp
+wget https://raw.githubusercontent.com/ODM2/ODM2/master/src/blank_schema_scripts/mysql/ODM2_for_MySQL.sql
+wget https://raw.githubusercontent.com/ODM2/ODM2/master/usecases/littlebearriver/sampledatabases/odm2_mysql/LBR_MySQL_SmallExample.sql
+
+cd $APP_PATH
+./docker_dev/scripts/wait-for-mysql.sh
+mysql -h$MYSQL_HOST -u$MYSQL_USER -p$MYSQL_PASSWORD -e "CREATE DATABASE ODM2;"
+mysql -h$MYSQL_HOST -u$MYSQL_USER -p$MYSQL_PASSWORD odm2 < $APP_PATH/tmp/ODM2_for_MySQL.sql
+# mysql -h$MYSQL_HOST -u$MYSQL_USER -p$MYSQL_PASSWORD < LBR_MySQL_SmallExample.sql
```
Comment (lsetiawan) [inline docker_dev/scripts/setup-mysql.sh]: This is giving me some 'ReferenceKeyID' Not found error of sorts.

Comment (emiliom) [reply docker_dev/scripts/setup-mysql.sh]: Thanks.

**Pull request merged**